src/store.js

```diff
@@ -388,6 +388,10 @@ class Store {
       .destroy(resource.get('_self'))
       .then(() => resource.set('isDeleted', true))
   }
+
+  request (url, options = {}) {
+    return this._adapter.request(url, options)
+  }
```

Comment:
Where do you see this used instead of Backbone's CRUD? Providing direct access to make http requests feels like it goes against the ideals of Backbone.Store but should help with transitioning. If something needed to make a direct request with the http-adapater could it just create a new instance of the http-adapter for itself?

Reply:
@jessewagstaff there are a few use cases where it is convenient over reinvoking a new instance of the http-adapter, as well as giving more fine grain options for a request. Within one of our grove apps, we have about 8 one-off requests, some which are in XMLHttpRequest, Fetch, or jQueryAjax. It really doesn't fit super well with the ideals of the backbone.store, but allows for a method via proxy to the http-adapter instance for convenience and ease of use. Ideally, this should help with the transition. Ultimately, once those requests are updated to use the same thing that is tested, we can repeal and replace this with something else. Baby steps 🙂 .